c-api/dotnet/src/Picky/Native/picky_status.cs

```diff
@@ -0,0 +1,8 @@
+namespace Devolutions.Picky.Native
+{
+    public enum picky_status
```

Comment:
nit: Is this generated? If so, no big deal. Otherwise, convention would be `PickyStatus` with values `Success` and `Failure`

Reply:
Yes, this is generated code from C header and it should not be used in idiomatic C# code but only when calling native functions (I'll write new C# enums when appropriate)